css/styles.css

```diff
@@ -996,4 +996,4 @@ p#modal-close {
     justify-content: center;
     align-items: center;
   }
-
+}
```

Comment:
- [ ] I didn't 😃find the specific element to make the inline comment, but I used that just to let me make an inline comment to you, so at the first section of the page I see that you did well at adding the required changes by the previous reviewer, but as stated by the previous reviewer please 😊try to correct the dimension to be similar to the Figma template you can increase the padding of the section to make the space below the section as you see there is a space below the social icons, so please try to increase the padding to make the space like the Figma Design 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/220186035-85bdd471-804b-41e4-98f9-80a1e31d2d1f.png)   |    ![image](https://user-images.githubusercontent.com/83360636/220185995-b6929f7c-a2a8-471a-b618-b8eefc3b8ade.png)  |

Reply:
Hi @osamaashraf6 
can you have another look at it now